rskj-core/src/main/java/co/rsk/peg/federation/FederationStorageProviderImpl.java

```diff
@@ -380,7 +381,10 @@ private void saveProposedFederation(ActivationConfig.ForBlock activations) {
             return;
         }
 
-        Integer formatVersion = Optional.of(proposedFederation.getFormatVersion()).orElse(null);
+        Integer formatVersion = proposedFederation != null ?
+            proposedFederation.getFormatVersion()
+            : null;
```

Comment:
You can still make use of Optional to simplify this
```suggestion
        Integer formatVersion = Optional.ofNullable(proposedFederation)
                                .map(Federation::getFormatVersion)
                                .orElse(null);
```

Reply:
done, thanks